bots/task/github.py

```diff
@@ -325,6 +299,17 @@ def commits(self, branch='master', since=None):
                 yield commit
                 count += 1
 
+def whitelist(api=GitHub()):
+    users = set()
+    page = 1
+    count = 100
+    while count == 100:
+        data = api.get("/contributors?page={0}&per_page={1}".format(page, count)) or [];
```

Comment:
I'd rather we limit this to the cockpit contributors team, rather than the full contributors list.  That's too many people, considering we're running arbitrary code with access to the cockpit bots' github token (which can do a lot of stuff).